artcommon/artcommonlib/logutil.py

```diff
@@ -55,6 +55,7 @@ def setup_logging(log_level: int, debug_log_path: str):
                         force=True)
 
     root_logger = logging.getLogger()
+    root_logger.setLevel(log_level)
```

Comment:
The log level is for stderr, which will be set at line 62.

Reply:
updated